addons/hr_expense/wizard/hr_expense_split.py

```diff
@@ -57,7 +57,7 @@ def _onchange_product_id(self):
         In case we switch to the product without taxes defined on it, taxes should be removed.
         Computed method won't be good for this purpose, as we don't want to recompute and reset taxes in case they are removed on purpose during splitting.
```

Comment:
@adrienpeiffer Would you mind updating the comment to match the new behavior? :smile: